.gitignore

```diff
@@ -52,3 +52,6 @@ jobs.sqlite
 
 # for -e pip packages
 src
+
+#tickets
+tickets/
```

Comment:
Create a generated/ directory and store tickets in a folder tickets there and ignore generated folder